index.html

```diff
@@ -148,10 +147,38 @@ <h2 class="presentation">Tonic</h2>
                 <li class="languaje">javascript</li>
             </ul>
              <div class="project">
-                <button  type="submit" class="button_project" onclick="window.location.href='https://github.com/Alondra87/Portfolio1'">See Project</button>
+                <button  class="button_project" onClick="openModal(3)" type="button">See Project</button>
             </div>
     </div></div>
 </section>
+ <!--section modal-->
+ <div class="modal" id="modal">
+    <div class="middle-sec">
+       <div class="mds22">
+            <h2 id="modal_title" class="presentation">Tonic</h2>
+            <button data-close-button class="close-button" type="button">&times;</button>
+            <ul class="list_middle" id="modal_middle">
+               <li class="canopy">CANOPY</li>
+               <li class="counter"><img src="./css/Counter.png" alt=""></li>
+               <li class="counter">Back End Dev</li>
+               <li class="counter"><img src="./css/Counter.png" alt=""></li>
+               <li class="counter">2015</li>
+           </ul>
+           <br>
+        <div class="mds12">  
+       <figure>
+           <img id="modal_snap" class="mod_snaps" src="./css/Snapshoot_Portfolio.svg" alt="Snapshoot_Portfolio">
+       </figure>
+       </div>   
+       <p class="description" id="modal_text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus, mollitia molestiae quos enim error ducimus, voluptatibus quidem optio maiores iste nulla totam iure voluptatum necessitatibus repellendus vero eos sunt aut.</p>
+           <ul id="modal_lang" class="list_middle2"></ul>
+         <div class="button_foot">
+              <a id="modal_live" class="button_project" href="https://github.com/Alondra87/Portfolio1" target="_blank" rel="noreferrer noopener" aria-label="this is an external link (opens in a new tab)">See Live <img src="./css/circlearrow.png" alt=""></a>
+               <a  id="modal_source" class="button_project" href="#" target="_blank" rel="noreferrer noopener" aria-label="this is an external link (opens in a new tab)">See Source <img src="./css/kitty.png" alt=""></a>
+ </div>
+ <div id="overlay"></div>
```

Comment:
You did great work until now, just some design stuff to make your design perfect.
- [ ] Kindly make your image take the full width to match the original design.
- [ ] Kindly be sure that your text doesn't go out of the design.
- [ ] Kindly make the tags above the two buttons.

Your design 
![image](https://user-images.githubusercontent.com/35535130/167918982-2fae0203-70f3-4bf4-91c7-3002d53621fd.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/167919065-d87cf9fc-db44-43b6-8e18-a47e37ca69f6.png)